front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -133,351 +152,437 @@
+          }
+
+          &.invalid {
+            background-image: url('data:image/svg+xml,%3Csvg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"%3E%3Ctitle%3EAssets/TrainList/InvalidTrain%3C/title%3E%3Cg fill-rule="evenodd"%3E%3Crect width="48" height="48" fill="%23FFEEED"/%3E%3Cpath d="m48 26v22h-22l22-22zm0-26-48 48v-22l26-26h22z" fill="%23FF6868" opacity=".1"/%3E%3C/g%3E%3C/svg%3E');
+            background-repeat: repeat;
+            box-shadow: inset 0 -0.063rem 0 0 rgba(0, 0, 0, 0.25);
+            .checkbox-title {
+              min-width: 0;
+            }
+            .checkbox-title::after {
+              display: none;
+            }
+            .rolling-stock {
+              min-width: max-content;
+              white-space: nowrap;
+              font-weight: 600;
+              color: var(--error80);
             }
+          }
 
-            // DURATION BAR
-            &.with-colored-border {
-              border-right: 3px solid var(--secondary);
-              padding-right: 0.25rem;
-              &.colored-border-0 {
-                border-color: rgba(var(--green-rgb), 0.75);
-              }
-              &.colored-border-1 {
-                border-color: rgba(var(--yellow-rgb), 0.75);
-              }
-              &.colored-border-2 {
-                border-color: rgba(var(--red-rgb), 0.5);
-              }
+          &.modified {
+            z-index: 3;
+          }
+
+          &:hover:not(.modified) {
+            .rolling-stock {
+              opacity: 0.3;
+            }
+            .action-buttons {
+              display: flex;
             }
           }
         }
+
         &:not(.with-details) {
-          .scenario-timetable-train-with-right-bar {
-            .scenario-timetable-train {
-              .scenario-timetable-train-body,
-              .scenario-timetable-train-tags {
-                display: none;
+          .scenario-timetable-train {
+            .more-info,
+            .scenario-timetable-train-tags {
+              display: none;
+            }
+          }
+        }
+
+        .filter-panel {
+          background-color: var(--ambiantB5);
+          padding: 0 1.438rem 0.25rem 0.313rem;
+          border-top: solid 0.063rem var(--white);
+          border-bottom: solid 0.063rem rgba(0, 0, 0, 0.25);
+          display: flex;
+          flex-direction: column;
+          align-items: end;
+
+          .grid-template {
+            display: grid;
+            grid-template-columns: 1fr 1fr;
+            gap: 0.625rem;
+          }
+
+          .close-filter {
+            transform: translateY(1rem);
+            z-index: 1;
+          }
+
+          .validity-filter {
+            display: flex;
+            margin-bottom: 1rem;
+            font-size: 0.8rem;
+
+            .options-btn {
+              padding: 0.375rem 0.625rem;
+            }
+          }
+
+          .compositions-code {
+            padding: 0.625rem 0.813rem 1rem 1rem;
```

Comment:
I don't think the spacing between the bottom of this component and the end of the filter panel should be handled partly by the padding bottom of this component. 
Mockup needs a 23px spacing, it should be directly by the parent with `.filter-panel` class.

Reply:
I pushed a solution where I adapted to the padding already existing on reusable components.